Src/Pages/CurrentProvider/CurrentProvider.tsx

```diff
@@ -0,0 +1,206 @@
+            variant={TextLabelVariants.LABEL}
+          >
+            {t('currentProvider.detail')}{' '}
+            <TextLabel
+              variant={TextLabelVariants.LABEL}
+              style={CurrentProviderStyle(themeColors).linkTextDetail}
+            >
+              {t('currentProvider.providerLink') + '.'}
+            </TextLabel>{' '}
+            {t('currentProvider.continueDetail')}
+          </TextLabel>
+
+          <TextLabel
+            variant={TextLabelVariants.INPUTLABEL}
+            style={{ marginTop: 18 }}
+          >
+            {t('currentProvider.optionTitle')}
+          </TextLabel>
+
+          <View style={CurrentProviderStyle(themeColors).selectionView}>
+            <TouchableOpacity
+              style={touchableView(selected?.createAccount)}
+              onPress={() => {
+                setSelected({ createAccount: true, changeProvider: false });
+              }}
+            >
+              <CreateAccountIcon
+                iconColor={iconStyle(selected?.createAccount)}
+              />
+
+              <TextLabel
+                style={titleStyle(selected?.createAccount)}
+                variant={TextLabelVariants.LABEL}
+              >
+                {t('currentProvider.createAccount')}
+              </TextLabel>
+            </TouchableOpacity>
+
+            <TouchableOpacity
+              style={touchableView(selected?.changeProvider)}
+              onPress={() => {
+                setSelected({ createAccount: false, changeProvider: true });
+              }}
+            >
+              <ChangeProviderIcon
+                iconColor={iconStyle(selected?.changeProvider)}
+              />
+
+              <TextLabel
+                style={titleStyle(selected?.changeProvider)}
+                variant={TextLabelVariants.LABEL}
+              >
+                {t('currentProvider.changeService')}
+              </TextLabel>
+            </TouchableOpacity>
+          </View>
+
+          <TouchableOpacity
+            style={CurrentProviderStyle(themeColors).providerInfo}
+            onPress={() => {}}
+          >
+            <ShowError
+              errorMessage={t('currentProvider.serviceProvider')}
+              styles={CurrentProviderStyle(themeColors).infoText}
+              colorCode={themeColors.currentProvider.titleUnSelected}
+              changeColor={true}
+            />
+          </TouchableOpacity>
+        </View>
+
+        <View style={styles(themeColors).buttonContainer}>
+          <ActionButton
+            disabled={!selected}
+            textStyle={[
+              CurrentProviderStyle(themeColors).sendText,
+              !selected && { color: themeColors.button.disableText },
+            ]}
+            buttonStyle={
+              !selected && {
+                backgroundColor: themeColors.button.disableBg,
+              }
+            }
+            title={t('buttonLabel.continue')}
+            onPress={async () => {
```

Comment:
this can also be single liner onPress={selected && handleSubmit}

Reply:
done